index.html

```diff
@@ -0,0 +1,286 @@
+        </p>
+      </div>
+      <div class="location">
+        <div class="dateinfo">Feb 15-17, 2023</div>
+        <div class="locate">@ Oakland Convention Center</div>
+      </div>
+    </section>
+  </div>
+
+  <div class="wrap main-program-wrap">
+    <section>
+      <h1 class="normal">Main program</h1>
+      <div class="guide-bar"></div>
+
+      <div class="desk-container">
+        <div class="program-list">
+          <div class="program-icon">
+            <img src="Images/program_icon_01.png" alt="">
+          </div>
+          <div class="program-sub-title">
+            <p>Conference</p>
+          </div>
+          <div class="program-sub-description">
+            <p>Listen to lectures from speakers from around the world and learn about the latest trends in the world.
+            </p>
+          </div>
+        </div>
+
+        <div class="program-list">
+          <div class="program-icon">
+            <img src="Images/program_icon_02.png" alt="">
+          </div>
+          <div class="program-sub-title">
+            <p>Hackathon</p>
+          </div>
+          <div class="program-sub-description">
+            <p>Join the largest challenge-driven in-person & virtual hackathon, co-located with DeveloperWeek 2023!</p>
+          </div>
+        </div>
+
+        <div class="program-list">
+          <div class="program-icon">
+            <img src="Images/program_icon_03.png" alt="">
+          </div>
+          <div class="program-sub-title">
+            <p>Open talks</p>
+          </div>
+          <div class="program-sub-description">
+            <p>We feature dozens of OPEN Talks.</p>
+          </div>
+        </div>
+
+        <div class="program-list">
+          <div class="program-icon">
+            <img src="Images/program_icon_04.png" alt="">
+          </div>
+          <div class="program-sub-title">
+            <p>Workshops</p>
+          </div>
+          <div class="program-sub-description">
+            <p>You can create your own creations using open source instead of just looking at them.</p>
+          </div>
+        </div>
+
+        <div class="program-list">
+          <div class="program-icon">
+            <img src="Images/program_icon_05.png" alt="">
+          </div>
+          <div class="program-sub-title">
+            <p>Awards</p>
+          </div>
+          <div class="program-sub-description">
+            <p>Recognizing outstanding design, engineering, and innovation in developer technology.</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <div class="show-all-program">
+      <a href="#">
+      <button class="button">Join DeveloperWeek</button>
+    </a>
+      <!-- <a href="#">View full program ></a> -->
+    </div>
+    <div class="show-all-program-desk display-none">
+      <a href="#">SEE THE WHOLE PROGRAM</a>
+    </div>
+  </div>
+
+  <div class="wrap main-speakers-wrap end-bar">
+    <section>
+      <h1 class="normal">Main speaker</h1>
```

Comment:
- [ ] I would recommend that you only use a single `<h1>` tag throughout your HTML file. So use `<h1>` for the main title of your website so that search engines will be able to know what the website is all about  👍🏽